src/lib.rs

```diff
@@ -74,41 +86,73 @@ use errors::*;
 
 mod diff;
 
-/// Assertions for a specific command
+/// Assertions for a specific command.
 #[derive(Debug)]
 pub struct Assert {
     cmd: Vec<String>,
-    expect_success: bool,
+    expect_success: Option<bool>,
```

Comment:
Yeah, you're right, this should be much more explicit. I'm also not sure if "expect success" is a good default, but I think I want to keep it for now. What do you think? Will the most common use case be to check something failed in a certain or that it succeeded?

I'm a big fan of fluent APIs and being explicit, that's why I added `.succeeds()` everywhere in the docs (and added `.and()`). It might be a good idea to leave it out in a few places.